apps/expo/src/app/post/[id].tsx

```diff
@@ -1,24 +1,22 @@
 import { SafeAreaView, Text, View } from "react-native";
-import { SplashScreen, Stack, useSearchParams } from "expo-router";
+import { Stack, useSearchParams } from "expo-router";
 
 import { api } from "../../utils/api";
 
-const Post: React.FC = () => {
+export default function PostPage() {
   const { id } = useSearchParams();
   if (!id || typeof id !== "string") throw new Error("unreachable");
   const { data } = api.post.byId.useQuery({ id });
 
-  if (!data) return <SplashScreen />;
```

Comment:
this loading state kinda sucked - maybe there should be a skeleton loader or something...